package/requirements.txt

```
@@ -10,3 +10,4 @@ pandas>=0.24
 sqlalchemy~=1.2
 strawberry-graphql>=0.99.0, <1.0
 networkx>=1.0
+orjson~=3.8.0
```

```suggestion
orjson~=3.8
```
No need to pin to the 3.8.x line (or probably to 3.x at all tbh... but that's how we normally do things at the moment).

---

Fixed in [49ba250](https://github.com/kedro-org/kedro-viz/pull/1087/commits/49ba25053c441c3ec90268d0cbe0d710fc2c4f7a).